schema.go

```diff
@@ -400,4 +435,327 @@ var metaSchemaData = []byte(`
         "exclusiveMinimum": [ "minimum" ]
     },
     "default": {}
-}`)
+}`),
+	draft06: []byte(`{
+		"$schema": "http://json-schema.org/draft-06/schema#",
```

Comment:
Schemas could be in files. Why there are such hardcoded strings?

Reply:
That map is used only in that file and is not supposed to be used of any other which is why it here